src/server/api/index.js

```diff
@@ -22,4 +25,43 @@ router.get('/platformcontext', (req, res) => {
   }
 });
 
+// LTI Launch params route
+router.get('/ltilaunch', (req, res) => {
+  try {
+    const action = process.env.RG_LTI_LAUNCH;
+    const method = 'POST';
+    const timestamp = Math.round(Date.now() / 1000);

```

Comment:
@kilometerskang Hey, I wonder if this would make a difference. I see in this article they say to use floor to get the UNIX timestamp: https://attacomsian.com/blog/unix-timestamp-javacript

const unixTime = Math.floor(Date.now() / 1000);

Maybe the LTI errors you show related to time is off by a second?

Reply:
tried, didn't work